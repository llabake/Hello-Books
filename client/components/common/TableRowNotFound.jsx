import React from 'react';
import PropTypes from 'prop-types';

const TableRowNotFound = ({ text }) => {
  return (
    <tr>
      <td>
      {text}
      </td>
    </tr>
  )
};

TableRowNotFound.propTypes = {
  text: PropTypes.string
};

export default TableRowNotFound;
