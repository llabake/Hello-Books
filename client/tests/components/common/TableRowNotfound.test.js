import React from 'react';

import TableRowNotFound from '../../../components/common/TableRowNotFound';

const wrapper = shallow(<TableRowNotFound />);

test('<TableRowNotFound /> mounts correctly', () => {
  expect(wrapper).toMatchSnapshot();
})

