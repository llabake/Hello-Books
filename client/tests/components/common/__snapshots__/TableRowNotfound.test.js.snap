// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableRowNotFound /> mounts correctly 1`] = `
<tr>
  <td />
</tr>
`;
